content/pt-br/docs/reference/glossary/container-runtime-interface.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Container runtime interface
+id: container-runtime-interface
+date: 2019-03-07
+full_link: /docs/concepts/architecture/cri
+short_description: >
+    O principal protocolo para a comunicação entre o kubelet e os agentes de execução de contêineres.
+
+aka:
+tags:
+- cri
+---
+
+O principal protocolo para a comunicação entre o kubelet e os agentes de execução de contêineres.
+
+<!--more-->
+
+A interface do agente de execução de contêiner do Kubernetes (CRI) define o principal protocolo [gRPC](https://grpc.io) para a comunicação entre os [componentes de cluster](/docs/concepts/overview/components/#node-components)
```

Comment:
A few bits here:
* How do you feel about moving "(CRI)" to right after "interface de execução de contêiner" here? I think it sounds better and is more faithful to the original text.
* The page linked in "componentes de cluster" is translated. You can prefix this address with `/pt-br`, so it links to the correct translated page.